packages/f-mega-modal/src/assets/scss/common.scss

```
@@ -0,0 +1,22 @@
+// Set $theme variable so that fozzie doesn't complain when vars are imported below
+// n.b. This variables isn't actually used (unless there are theme specific vars from fozzie)
+// Instead, the theme is handled via a data-attribute on the component (via the `@mixin theme()` below)
+$theme: 'je' !default;
+
+@import '~@justeat/f-utils/src/scss/functions/index';
+@import '~@justeat/f-utils/src/scss/mixins/index';
+@import '~include-media/dist/include-media'; // Cleaner media query declarations – http://include-media.com
+@import '~@justeat/fozzie/src/scss/settings/variables';
+@import '~@justeat/fozzie/src/scss/base/reset';
+@import '~@justeat/fozzie-colour-palette/src/scss/';
+
+/**
+ * Component Variables
+ */
+// $megaModal-background-color: $brand--orange;
+
+@mixin theme($type) {
+    [data-theme="#{$type}"] & {
```

Would be good to namespace this mixin, as [has been done for other components](https://github.com/justeat/fozzie-components/blob/master/packages/f-card/src/assets/scss/common.scss#L19) in [this PR](https://github.com/justeat/fozzie-components/pull/255)

```suggestion
    [data-theme-mega-modal="#{$type}"] & {
```

---

As mentioned in a comment above I'm going to remove this mixin as it's not used in this component.